components/script/devtools.rs

```diff
@@ -68,7 +68,11 @@ pub fn handle_evaluate_js(global: &GlobalRef, eval: String, reply: IpcSender<Eva
 }
 
 pub fn handle_get_root_node(context: &BrowsingContext, pipeline: PipelineId, reply: IpcSender<NodeInfo>) {
-    let context = context.find(pipeline).unwrap();
+    let context = match context.find(pipeline) {
+        Some(found_context) => found_context,
+        None => return warn!("No root node found for pipeline id {}", pipeline)
```

Comment:
@craftytrickster Maybe you can make `reply` a `IpcSender<Result<NodeInfo, SomeErrorType>>`? (I am not an expert on that, but it looks like no other places are using this message now so I think it is safe to change its type)